config/routes.rb

```diff
@@ -112,9 +112,9 @@
       resource :category_assessments, only: %i[create show]
 
       resource :start, only: %i[new]
-      resource :moving_requirements, only: %i[new create]
+      resource :moving_requirements, only: %i[new create update]
```

Comment:
```suggestion
      resource :moving_requirements, only: %i[new create]
```

Reply:
So my understanding here is we have a rendered form (new) and a submitted (post) to a form create.